Create ui-c8y-1023-14-178-device-management-can-t-show-custom-widgets.md (#4855)
SoftwareAG/c8y-docs: content/change-logs/device-management/ui-c8y-1023-14-178-device-management-can-t-show-custom-widgets.md

---
date: ""

> title: Custom widgets now shown correctly in device dashboards

Reviewer: ```suggestion
title: Legacy dashboards no longer shown in the Device Management application
```

product_area: Device management & connectivity
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component--KIsStyzM
    label: Device Management app
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: DM-5725
version: 1023.14.178
---

> Dashboards created without an explicit application association (legacy dashboards lacking the c8y_AppliedToApplications fragment) were incorrectly included when resolving device dashboards in the Device Management application. The Device Management app now only shows dashboards explicitly bound to it, preventing unintended display of dashboards that belong to other applications or were created before application-scoped dashboards were introduced.

Reviewer 2: ```suggestion
Dashboards created without an explicit application association (legacy dashboards lacking the `c8y_AppliedToApplications` fragment) were incorrectly included when resolving device dashboards in the Device Management application. The Device Management application now only shows dashboards explicitly bound to it, preventing unintended display of dashboards that belong to other applications or were created before application-scoped dashboards were introduced.
```

> In the Device Management application, custom widgets configured for specific application dashboards were not displayed because legacy dashboards without an explicit application association were being included in the dashboard resolution process, which conflicted with app-specific dashboards. The Device Management app now filters device dashboards strictly to those explicitly bound to the current application. Legacy dashboards that lack a c8y_AppliedToApplications fragment are no longer included, ensuring custom widgets appear as expected.

Reviewer: ```suggestion
Dashboards created without an explicit application association (legacy dashboards lacking the c8y_AppliedToApplications fragment) were incorrectly included when resolving device dashboards in the Device Management application. The Device Management app now only shows dashboards explicitly bound to it, preventing unintended display of dashboards that belong to other applications or were created before application-scoped dashboards were introduced.
```

---
date: ""

> title: Device management can t show custom widgets

Reviewer: ```suggestion
title: Custom widgets now shown correctly in device dashboards
```

product_area: Device management & connectivity
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component--KIsStyzM
    label: Device Management app
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: DM-5725
version: 1023.14.178
---
# Backport

This will backport the following commits from `develop` to
`release/y2026`:
- [fix(Device Management): [DM-5725] Device management can t show custom
widgets](https://github.com/Cumulocity-IoT/cumulocity-ui/pull/12225)

<!--- Backport version: 9.5.1 -->

### Questions ?
Please refer to the [Backport tool
documentation](https://github.com/sorenlouv/backport)

[DM-5725]:
https://cumulocity.atlassian.net/browse/DM-5725?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

> Co-authored-by: Bozhidar Dedov <95280800+bozhidardedov@users.noreply.github.com>

Reviewer: ```suggestion
In the Device Management application, custom widgets configured for specific application dashboards were not displayed because legacy dashboards without an explicit application association were being included in the dashboard resolution process, which conflicted with app-specific dashboards. The Device Management app now filters device dashboards strictly to those explicitly bound to the current application. Legacy dashboards that lack a c8y_AppliedToApplications fragment are no longer included, ensuring custom widgets appear as expected.
```
